README.md

```diff
@@ -1,38 +1,107 @@
 # bcrpc
-Tiny Bitcoin RPC wrapper for node.js
+Tiny Bitcoin RPC wrapper for Node.js.
 
 ## Usage
-Each rpc function is exposed as a function in an instantiated RpcAgent. The first arguments are parsed as array of arguments, as expected by that specific rpc function, see [Bitcoin-core RPC Docs](https://bitcoincore.org/en/doc). The last argument is an optional callback function, if no callback is provided, it will return a promise.
+Each rpc function is exposed as a function in an instantiated RpcAgent. The first arguments are parsed as array of arguments, as expected by that specific rpc function, see [Bitcoin core RPC Docs](https://bitcoincore.org/en/doc). The last argument is an optional callback function, if no callback is provided, it will return a promise. 
 
-```
-const RpcAgent = require('./rpc');
+The following program can be saved and and run in the same directory as bcrpc. It will print out the latest block number along with its hash. Change username and password to those from ```~/.bitcoin/bitcoin.conf``` and also change the port number to 18332 if you are running Bitcoin on testnet.
```

Comment:
`~/bitcoin` is for linux installations only; just say "the `bitcoin.conf` file in the Bitcoin data directory"?